chain/chain/tests/simple_chain.rs

```diff
@@ -138,20 +139,61 @@ fn blocks_at_height() {
     init_test_logger();
     let (mut chain, _, signer) = setup();
     let genesis = chain.get_block_by_height(0).unwrap();
-    let b_1 = Block::empty_with_height(genesis, 1, &*signer);
-    let b_2 = Block::empty_with_height(&b_1, 2, &*signer);
-    let b_3 = Block::empty_with_height(&b_2, 3, &*signer);
+    let b_1 = Block::empty_with_timestamp(
```

Comment:
We should have `let utc_now = Utc::now()` instead of recomputing it. To make it debuggable, currently, if I put a breakpoint and stop a program for a couple of seconds it will make `Utc::now()` be very different.

Ideally, we want timer to be injectable into the production code so that the in unit tests we can have a fake timer, instead of relying on the system timer. It might be too much work to do now, but we should clean it up in the future.

https://github.com/nearprotocol/nearcore/issues/1770